src/App.jsx

```diff
@@ -136,16 +148,17 @@ class App extends React.Component {
     const cocktailNumber = this.state.intersection.length;
     return (
       <div>
+        <div>
+          {this.state.showPopup ? <PopupAge closePopupEnter={this.togglePopup} closePopupExit={this.exitSite} /> : null }
```

Comment:
never `: null` ;)

try:

    { this.state.showPopup && <PopupAge closePopupEnter={this.togglePopup} closePopupExit={this.exitSite} /> }

Reply:
Thanks, changes done ! :)